modules/updating-hosted-cluster.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * updates/updating_a_cluster/updating-hosted-control-planes.adoc
+// * hosted_control_planes/hcp-updating.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="updating-hosted-cluster_{context}"]
+= Updating the hosted cluster for hosted control planes
+
+On hosted control planes, you can update your version of {product-title} by updating the hosted cluster.
```

Comment:
Did we explain all these entities briefly somewhere in our doc: hosted control planes, hosted cluster, node pools, etc?

Reply:
yes. Here https://docs.openshift.com/container-platform/4.15/hosted_control_planes/index.html#hosted-control-planes-concepts-personas_hcp-overview

Reply:
But it doesn't cover node pools. I will cover that in a follow up PR

Reply:
https://github.com/openshift/openshift-docs/pull/76215 -- node pools definition ; PR is merged